integration-tests/actions/vrfv2plus/vrfv2plus_steps.go

```diff
@@ -43,90 +48,94 @@ var (
 	ErrGetPrimaryKey            = "error getting primary ETH key address"
 	ErrRestartCLNode            = "error restarting CL node"
 	ErrWaitTXsComplete          = "error waiting for TXs to complete"
+
+	ErrRequestRandomness             = "error requesting randomness"
+	ErrWaitRandomWordsRequestedEvent = "error waiting for RandomWordsRequested event"
+
+	ErrWaitRandomWordsFulfilledEvent = "error waiting for RandomWordsFulfilled event"
 )
 
 func DeployVRFV2PlusContracts(
 	contractDeployer contracts.ContractDeployer,
 	chainClient blockchain.EVMClient,
+	consumerContractsAmount int,
 ) (*VRFV2PlusContracts, error) {
 	bhs, err := contractDeployer.DeployBlockhashStore()
 	if err != nil {
 		return nil, errors.Wrap(err, ErrDeployBlockHashStore)
 	}
+	//err = chainClient.WaitForEvents()
+	if err != nil {
+		return nil, errors.Wrap(err, ErrWaitTXsComplete)
+	}
 	coordinator, err := contractDeployer.DeployVRFCoordinatorV2Plus(bhs.Address())
 	if err != nil {
 		return nil, errors.Wrap(err, ErrDeployCoordinator)
 	}
-	loadTestConsumer, err := contractDeployer.DeployVRFv2PlusLoadTestConsumer(coordinator.Address())
+	//err = chainClient.WaitForEvents()
+	if err != nil {
```

Comment:
if line 75 is not needed this errcheck is duplicated

Reply:
enabled back the waitforevents